modules/hcp-aws-create-secret-s3.adoc

```diff
@@ -38,21 +46,29 @@ $ echo '{
             "Effect": "Allow",
             "Principal": "*",
             "Action": "s3:GetObject",
-            "Resource": "arn:aws:s3:::<bucket_name>/*" <1>
+            "Resource": "arn:aws:s3:::<bucket_name>/*"
         }
     ]
 }' | envsubst > policy.json
 ----
 +
-<1> Replace `<bucket_name>` with the name of the S3 bucket you are creating.
+where:
++
+--
+<bucket_name>:: Is the name of the S3 bucket you are creating.
```

Comment:
I assume the double colons are expected (as other examples in the PR), is that the case?